src/js/ripple/amount.js

```diff
@@ -176,13 +189,7 @@ Amount.prototype.ratio_human = function(denominator, opts) {
   opts = extend({ }, opts);
 
   var numerator = this;
-
-  if (typeof denominator === 'number' && parseInt(denominator, 10) === denominator) {
-    // Special handling of integer arguments
-    denominator = Amount.from_json(String(denominator) + '.0');
-  } else {
-    denominator = Amount.from_json(denominator);
-  }
+  denominator = Amount.from_json(noIntegers(denominator));
```

Comment:
I mean, it seems like from_json isn't desired, so it's not really wanting to interpret it as a native amount. But then later, I see code like equals() which will just use from_json. Hrmmm. Not really consistent semantics.

I can't recall off hand what from_json does for Number values.